src/types/Ctype.ts

```diff
@@ -25,20 +33,19 @@ export interface IClaimInput {
   title: string
   description?: string
   type: string
+  owner?: string
 }
 
 export class CType implements ICType {
-  public static fromObject(obj: ICType): CType {
+  public static fromObject(obj: CTypeMetadata): CType {
     const newCtype = Object.create(CType.prototype)
     return Object.assign(newCtype, obj)
   }
 
-  public metaData: {
-    author: string
-  }
   public readonly cType: sdk.ICType
-
+  public metadata: IMetadata
+  public ctypeHash: sdk.ICType['hash']
   public getPropertyTitle(propertyName: string) {
-    return this.cType.metadata.properties[propertyName].title.default
+    return propertyName // Need to fix. I need to take the value of the key.
```

Comment:
This needs to be fixed, before we merge

Reply:
I reverted it to the previous state, might not work as we want it to be, I would need your input.

Reply:
removed it and moved the logic to the original calling functions `getCtypePropertyTitle` in 

src/components/SelectAttestedClaim/SelectAttestedClaim.tsx
src/components/AttestedClaimVerificationView/AttestedClaimVerificationView.tsx

with an additional check whether the property exists on the metadata:

`cType && cType.metadata.properties[propertyName].title.default ?
      cType.metadata.properties[propertyName].title.default : propertyName`